lib/clis/wolkenkit/health/healthCommand.ts

```diff
@@ -0,0 +1,88 @@
+import { buntstift } from 'buntstift';
+import { Client } from '../../../apis/getHealth/http/v2/Client';
+import { Command } from 'command-line-interface';
+import { HealthOptions } from './HealthOptions';
+import { validatePort } from '../dev/validatePort';
+
+const healthCommand = function (): Command<HealthOptions> {
+  return {
+    name: 'health',
+    description: 'Run a health check against a process.',
+
+    optionDefinitions: [
+      {
+        name: 'protocol',
+        description: 'set the protocol',
+        parameterName: 'protocol',
+        type: 'string',
+        isRequired: false,
+        defaultValue: 'http'
+      },
+      {
+        name: 'host-name',
+        description: 'set the host name of the server',
```

Comment:
```suggestion
        description: 'set the host name',
```

(If we don't specify "of the server"  for the protocol, we don't need it here as well … and I assume it's obvious whose host  name and protocol are needed)